spaceship/lib/spaceship/connect_api/client.rb

```diff
@@ -19,10 +19,229 @@ def self.hostname
       end
 
       def build_url(path: nil, filter: nil, includes: nil, limit: nil, sort: nil)
```

Comment:
Any particular reason we aren't using Faraday's built in query param serializer?

Reply:
I believe it would be something like:
```ruby
response = request(:post, url) do |r|
  r.params["include"] = includes if incluces
  r.params["filter"] = filters if filters
  .
  .
  .
end
```

If the filter key/val dont serialize correctly by default, I think it can be overridden.

Reply:
@snatchev Probably because I didn't know that existed 😉